cloudslang-runtime/src/main/java/io/cloudslang/lang/runtime/steps/ActionSteps.java

```diff
@@ -63,10 +60,16 @@ public void doAction(@Param(ScoreLangConstants.RUN_ENV) RunEnvironment runEnv,
 
         Map<String, Serializable> returnValue = new HashMap<>();
         Map<String, Serializable> callArguments = runEnv.removeCallArguments();
+        Map<String, Serializable> callArgumentsDeepCopy = new HashMap<>();
+
+        for (Map.Entry<String, Serializable> entry : callArguments.entrySet()) {
+            callArgumentsDeepCopy.put(new String(entry.getKey().toString()),  SerializationUtils.clone(entry.getValue()));
```

Comment:
you can use simply `entry.getKey()` here instead of `new String(entry.getKey().toString()` because of diamond operator (`Map.Entry<String, Serializable> entry`)